autoload/shiny.vim

```diff
@@ -71,24 +88,33 @@ function! s:flash_and_paste(target) abort
   call s:flash(patterns, s:vim_shiny_hi_paste)
 endfunction
 
-function! s:flash(patterns, group) abort
-  let i = 0
-  for p in a:patterns
-    call s:Highlight.highlight('ShinyFlash' . i, a:group, p, 1)
-    let i += 1
+function! s:_flash(patterns, group) abort
+  let bg = &background == 'dark' ? 'dark' : 'light'
```

Comment:
**[vint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Use robust operators `==#` or `==?` instead of `==` (see Google VimScript Style Guide (Matching))